CoreJava/.classpath

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You can exclude these . file from git(add these files into .gitignore) so these will not include in git changes